CDCgov/prime-reportstream
# 14102: disable dedupe detection parameter for the UP

This PR disables the dedupe detection code for the UP as it's broken and can cause no items to get routed

Test Steps:
1. Send a HL7 message with the `allowDuplicate` header set and verify that the report gets routed

## Changes
- Disables the duplicate detection in the Universal pipeline receiver

## Checklist

### Testing

### Process

## Linked Issues
- #14102

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #14103

## Specific Security-related subjects a reviewer should pay specific attention to
